src/App.js

```diff
@@ -1,30 +1,48 @@
-import React, { Fragment, useEffect } from 'react'
+import React, { useState } from 'react'
+import { IntlProvider } from 'react-intl'
 import { BrowserRouter, Route, Switch } from 'react-router-dom'
-
 import { Provider } from 'react-redux'
-import store from './store'
-import SignInPage from './components/pages/SignInPage'
-import SignUpPage from './components/pages/SignUpPage'
-import ProtectedRoutes from './components/routes/ProtectedRoutes'
-import Home from './components/Home'
+import { PersistGate } from 'redux-persist/integration/react'
+import { store, persistor } from 'store'
+import flatten from 'flat'
+import ForgotPasswordPage from 'components/pages/ForgotPasswordPage'
```

Comment:
Pages should be imported using code split, you can import `lazy` from react:
```suggestion
const ForgotPasswordPage = lazy(() => import('pages/ForgotPasswordPage'));
```